CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.4.1]
+
+### Fixed
+
+- feat: Inject MetaMask wallet name for prod and flask release ([#35](https://github.com/MetaMask/solana-wallet-standard/pull/35))
```

Comment:
we should indicate why this patch was needed and whether the last release should be deprecated